src/main/java/com/neprozorro/rest/MockResource.java

```diff
@@ -0,0 +1,28 @@
+package com.neprozorro.rest;
+
+import com.neprozorro.service.MockService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController()
+@RequiredArgsConstructor
+public class MockResource { //Todo for remove
```

Comment:
comment suggests removing but I'm not sure if the comment should be here. Don't see a problem with having an additional controller

Reply:
Ok.